applications/nrf_desktop/src/modules/dvfs.c

```diff
@@ -35,32 +35,50 @@ enum state {
 
 static enum state module_state = STATE_DISABLED;
 
+#define FOR_DVFS_STATE_LIST(MACRO, ...)				\
```

Comment:
I would recommend prefixing all these macros so that it is clear that they are not general macros (like IF_ENABLED) but they are module-specific ones. For example:

DESKTOP_DVFS_LIST_DVFS_STATE
DESKTOP_DVFS_DEFINE_ENUM
(...)

I am also in favour of putting verb at the end in function/macros, but it seems that we do not follow this approach in the nRF Desktop code base:


DESKTOP_DVFS_STATE_LIST
DESKTOP_DVFS_ENUM_DEFINE
(...)

Reply:
>> I would recommend prefixing all these macros so that it is clear that they are not general macros (like IF_ENABLED) but they are module-specific ones. For example:

We do not do it with all the functions we have in module. This is c file.  Also it creates a feeling that all those macros look similar for me so the overall code is less clear.

>> I am also in favour of putting verb at the end in function/macros, but it seems that we do not follow this approach in the nRF Desktop code base:

I can change the order but is there any guide that it should be done like that? IMO  verb at beginning looks better.

Reply:
>> I can change the order but is there any guide that it should be done like that? IMO verb at beginning looks better.

Yes, there is a guide for that: 
https://nordicsemi.atlassian.net/wiki/spaces/QPDA/pages/145118051/Nordic+C+Coding+Standard
6.1.9 ... this "noun-verb" word ordering is recommended. ...

We will try to keep up to that next time.